yt_dlp/extractor/thisoldhouse.py

```diff
@@ -15,6 +16,11 @@ class ThisOldHouseIE(InfoExtractor):
             'description': 'In the workshop, Tom Silva and Kevin O\'Connor build a storage bench for an entryway.',
             'timestamp': 1442548800,
             'upload_date': '20150918',
+            'duration': 674,
+            'view_count': int,
+            'average_rating': 0,
```

Comment:
this is actually 0 and not lack of data?

Reply:
It is actually 0.
Maybe bcoz the video is old ig.